cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -15,13 +15,121 @@
 # ~~~
 
 include(CMakeDependentOption)
+include(CreateBazelConfig)
 
 # The default list of libraries to build. These can be overridden by the user by
 # passing a comma-separated list, i.e
 # `-DGOOGLE_CLOUD_CPP_ENABLE=spanner,storage`.
 set(GOOGLE_CLOUD_CPP_LEGACY_FEATURES
     "bigtable;bigquery;iam;logging;pubsub;spanner;storage")
 
+set(GOOGLE_CLOUD_CPP_EXPERIMENTAL_FEATURES
+    # cmake-format: sorted
+    # Introduced circa 2022-08-17
+    "apikeys"
+    # This is WIP, it needs a number of hand-crafted APIs.
+    "pubsublite")
+
+set(GOOGLE_CLOUD_CPP_TRANSITION_FEATURES # cmake-format: sorted
+    # Promoted to GA circa 2022-10-12
+    "batch")
+
+set(GOOGLE_CLOUD_CPP_GA_FEATURES
```

Comment:
are you sure you want to call these things `FEATURES` and not `LIBRARIES` ?

I think I would expect to see `storage-grpc`, `grafeas`, `generator`, `opentelemetry` also listed as features.

Reply:
Fixed.